### DISCUSSION TITLE
Creating a copy to clipboard button within PnP Search Results webpart

### DISCUSSION THREAD

**Discussion by JazzyLeopard** [microsoft-search/pnp-modern-search#3413] [Ideas]
Hi guys, 

I understood that custom JS is not possible within the Search Results webpart because of security risks. 
Is there any other way to create a copy to clipboard button within the webpart? 

Do keep in mind that I'm a beginner :D 

Thanks in advance!

**Comment by wobba** [comment]
You need to do extensibility development to achieve this. See https://microsoft-search.github.io/pnp-modern-search/extensibility/custom_web_component

Which is not exactly beginner work unfortunately. What of the results do you want to copy to clipboard?

**Comment by JazzyLeopard** [comment]
![Screenshot 2023-12-17 001616](https://github.com/microsoft-search/pnp-modern-search/assets/153996776/5d81a781-88f6-4bf1-a5ef-472649b42114)

In the screenshot attached, I want to be able to copy the text in the results to clipboard to make it easier for users.

**Reply by wobba** [reply]
If someone create a component for this we would accept the contribution for sure.
